src/collapse.ts

```diff
@@ -0,0 +1,95 @@
+import assert from 'assert';
+
+export default class Collapse {
```

Comment:
This class needs some documentation: what is it doing, & the fact that it was adapted (copied?) from bootstrap